source/mobile/deploy-mobile-apps-using-emm-provider.rst

```diff
@@ -0,0 +1,132 @@
+Deploying Mobile Apps Using an EMM Provider
+===========================================
+
+An EMM Provider builds `Enterprise Mobile Management software <https://en.wikipedia.org/wiki/Enterprise_mobility_management>`__ that helps enterprise teams manage secure mobile endpoints with managed app configuration. 
```

Comment:
```suggestion
An EMM Provider `Enterprise Mobile Management software <https://en.wikipedia.org/wiki/Enterprise_mobility_management>`__ that helps enterprise teams manage secure mobile endpoints with managed app configuration. 
```

Reply:
Rewrote this sentence for clarity. Thanks!